### ISSUE TITLE
mime type problem in Android R (11, API 30)

### ISSUE THREAD

**Issue by jp-eagle** [Dhaval2404/ImagePicker#130]
I got issue in Android R when I tried to set mime type as video and audio. using galleryMimeTypes().

  the below code, works well in Android Q (10, API 29) and below, it is able to show video and audio files only. but it won't work in Android R (11, API 30), it shows photos only, not doesn't show video and audio files.
``` javascript
            ImagePicker.Companion.with(this)
                    .galleryOnly()
                    .galleryMimeTypes(new String[]{"video/*", "audio/*"})
                    .start();
```

the part of Manifest is below,
```
<uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />

android:requestLegacyExternalStorage="true"
```

the part of gradle file is below, I didn't add florent37:inline-activity-result-kotlin

```
implementation 'com.github.dhaval2404:imagepicker:1.7.5'
compileSdkVersion 30
minSdkVersion 24
targetSdkVersion 30
buildToolsVersion '30.0.2'

```

**Comment by Dhaval2404** [comment]
Haha, I didn't know that the ImagePicker can also be used to Pick Audio & Video files. Just gave it a try on Android 11 and It seems to be working fine. I have tested in Pixel 2 Emulator(API 30). I will check this on other devices just to verify the issue.

Sorry for the late reply.

**Comment by Dhaval2404** [comment]
I have tested in Pixel 4a (with Android 11, API 30) and It's working fine. Please let me know the device to reproduce the issue. Looking forward to your reply. Thank you!

**Comment by jp-eagle** [comment]
> I have tested in Pixel 4a (with Android 11, API 30) and It's working fine. Please let me know the device to reproduce the issue. Looking forward to your reply. Thank you!

Thanks for the reply. I don't have a physical phone with Android 11. I use Google Play Intel x86 Atom_64 System Image to test. it doesn't pick audio and video file.

**Comment by Dhaval2404** [comment]
@jp-eagle Today, I will check on Google Play Intel x86 Atom_64 System Image. 

Thank you so much for your reply.

**Comment by jp-eagle** [comment]
any finding about this?

**Comment by joaodforce** [comment]
<img width="692" alt="image" src="https://user-images.githubusercontent.com/9410916/115032706-fc9bf500-9e9f-11eb-9177-9c29874b5823.png">

I believe this is the cause of the issue.

**Comment by joaodforce** [comment]
I managed to fix the compatibility issue by rolling back my SDK Target to 28, but this need to be fixed so it can be used with future SDKs in the future.

**Comment by Dhaval2404** [comment]
@jp-eagle Issue is resolved in ImagePicker v2.0. Please give it a try.

![image](https://user-images.githubusercontent.com/7314181/119261240-24475100-bbf4-11eb-9046-463f2fd07303.png)

<div>
<img src="https://user-images.githubusercontent.com/7314181/119261005-f0b7f700-bbf2-11eb-937e-193702434aaa.png" width="280">

<img src="https://user-images.githubusercontent.com/7314181/119261015-fad9f580-bbf2-11eb-9920-60871dd85d36.png" width="280">
</div>

**Comment by qadirsuh** [comment]
Hehehe.. I was finding answer to this question too. 
Does this awesome library can work to pick any kind of file i.e audio, video, image :)
Also want to know can we select and pick multiple files in one go.
